cluster-cm/src/main/java/com/sequenceiq/cloudbreak/cm/ClouderaManagerModificationService.java

```diff
@@ -128,13 +130,17 @@ public void initApiClient() throws ClusterClientInitException {
     @Override
     public List<String> upscaleCluster(HostGroup hostGroup, Collection<InstanceMetaData> instanceMetaDatas)
             throws CloudbreakException {
+        LOGGER.debug("Invoking CM requests for upscale");
+        int hostCount = -1;
```

Comment:
this is a bit odd. I think it would make sense to use Integer and we can log `null` in that case, or use `Optional<Integer>` and use `isPresent` when logging